src/data/gitmojis.json

```
@@ -311,7 +311,7 @@
       "emoji": "💡",
       "entity": "&#128161;",
       "code": ":bulb:",
-      "description": "Documenting source code.",
+      "description": "Adding or updating comments in source code",
```

The majority of the emojis has the period, what do you think about keeping the dot in the end?

```suggestion
      "description": "Adding or updating comments in source code.",
```

---

I did not notice. Yes of course !